src/tests/multihost/alltests/test_services.py

```diff
@@ -42,17 +46,21 @@ def test_0002_1736796(self, multihost, localusers):
          cause files domain entries to be qualified, this can break sudo access
         :id: 4b7bdeff-51ba-46ed-b8e1-0685515b87a0
         """
+        if not have_files_provider(multihost):
+            pytest.skip("Files Provider support isn't available, skipping")
         users = localusers
         for user in users.keys():
             allow_sudo = '%s    ALL=(ALL) NOPASSWD:ALL' % user
             sudoers_file = '/etc/sudoers.d/%s' % user
             multihost.client[0].put_file_contents(sudoers_file, allow_sudo)
         tools = sssdTools(multihost.client[0])
         sssd_params = {'default_domain_suffix': 'foo',
-                       'domains': 'LOCAL'}
+                        'domains': 'LOCAL'}
         tools.sssd_conf('sssd', sssd_params)
         domain_section = 'domain/LOCAL'
-        domain_params = {'id_provider': 'files'}
+        domain_params = {'id_provider': 'proxy',
+                        'proxy_lib_name': 'files',
+                        'proxy_pam_target': 'sssd-shadowutils'}
```

Comment:
What sense does it make: to skip-if-files-not-built **and** replace files provider with proxy provider?

Reply:
The same question everywhere.

Reply:
Look, if test only makes sense for "files provider" (I don't know if this is the case here) then you just skip it if files provider isn't supported.

If test makes sense for both 'files' and 'proxy::files' provider then make test run it with both providers (where supported).

Finally, if 'files' provider was used just as auxiliary domain, then replace it with 'proxy::files'.

Reply:
Skipif should be for the tests that are files provider specific.
Proxy provider can conditionally replace files provider for tests that are not specific for files provider.
We are not going to duplicate legacy tests to run on both providers.

Reply:
Main point is that this patch doesn't make sense at all.

Reply:
If patch does not make sense, then it's either fail for coming builds or we need to skip or update it.

Reply:
@alexey-tikhonov, can you **please** explain to me why this patch does not make any sense?

Reply:
https://github.com/SSSD/sssd/pull/7176/files#r1486026611

Reply:
If you test files provider (i.e. keep using 'id_provider=files') then you skip the test if 'files provider' support isn't built.
But if you replaced 'id_providers=files' with 'id_provider=proxy' then what sense does it make to condition test execution on availability of 'files provider'?

Reply:
yes, I agreed.  I am ready to update those test cases, so that will be skipped. But that does not mean the whole patch does not make any sense. If we have tests cases, either we need to update or skip it.

Reply:
A 'patch' is a change you are proposing.

Reply:
A 'patch' is **not** original test.